SignalMessaging/environment/OWSSounds.m

```diff
@@ -74,8 +74,8 @@ - (instancetype)initWithStorageManager:(TSStorageManager *)storageManager
         @(OWSSound_Keys),
         @(OWSSound_Popcorn),
         @(OWSSound_Pulse),
-        @(OWSSound_Synth),
         @(OWSSound_ClassicNotification),
+        @(OWSSound_Synth),
```

Comment:
I thought the bottom of the list was maybe an easier place to find the "Classic" notification, but now that it's called "Signal Classic" (right next to "Synth") it just looked like we had mis-sorted alphabetically.

Reply:
Maybe we should rename `OWSSound_ClassicNotification` to  `OWSSound_SignalClassic` to make it more obvious why the sort is in this order.

Reply:
Done